WcaOnRails/app/models/round.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class Round < ApplicationRecord
+  belongs_to :competition_event
+  has_one :event, through: :competition_event
+  belongs_to :format
+
+  MAX_NUMBER_OF_ROUNDS_PER_EVENT = 4
```

Comment:
Ok, I'm tempted to wait until we decide on the new name for `nth_round`, and then we can name this variable accordingly (`MAX_NTH_ROUND`, for instance).